src/eradius_proxy.erl

```diff
@@ -66,13 +77,21 @@ validate_arguments(Args) ->
             {true, [{default_route, DefaultRoute}, {options, Options}, {routes, NewRoutes}]}
     end.
 
-compile_routes(Routes) ->
-    RoutesOpts = lists:map(fun ({Name, Relay}) ->
+compile_routes(false) ->
```

Comment:
`undefined` more better for perception as `false`

Reply:
this is fixed as well